examples/vesting-wallet/tests/vesting-wallet.rs

```diff
@@ -0,0 +1,477 @@
+#![cfg(feature = "e2e")]
+
+use abi::VestingWallet;
+use alloy::{
+    eips::BlockId,
+    primitives::{Address, U256},
+    providers::Provider,
+    sol,
+};
+use e2e::{
+    receipt, send, watch, Account, EventExt, Panic, PanicCode, ReceiptExt,
+    Revert,
+};
+use mock::{erc20, erc20::ERC20Mock};
+
+use crate::VestingWalletExample::constructorCall;
+
+mod abi;
+mod mock;
+
+sol!("src/constructor.sol");
+
+const BALANCE: u64 = 1000;
+const DURATION: u64 = 365 * 86400; // 1 year
+
+fn ctr(
+    beneficiary: Address,
+    start_timestamp: u64,
+    duration_seconds: u64,
+) -> constructorCall {
+    constructorCall {
+        beneficiary,
+        startTimestamp: start_timestamp,
+        durationSeconds: duration_seconds,
+    }
+}
+
+async fn block_timestamp(account: &Account) -> eyre::Result<u64> {
+    let timestamp = account
+        .wallet
+        .get_block(
+            BlockId::latest(),
+            alloy::rpc::types::BlockTransactionsKind::Full,
```

Comment:
Why not import `use alloy::rpc::types::BlockTransactionKind;`?

Reply:
It is only used here and nowhere else, but will import it as suggested